www/package-lock.json

```diff
@@ -0,0 +1,8650 @@
+{
```

Comment:
I'm not sure but this look off to me @huytd

Reply:
Let's ignore the `package-lock.json` @conanak99

Reply:
Commited 😄